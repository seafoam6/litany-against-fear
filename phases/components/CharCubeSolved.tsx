import React from "react";

interface Props {
  solution: string;
}

const CharCubeSolved: React.FC<Props> = ({ solution }) => {
  return (
    <>
      <div>{solution}</div>
      <style jsx>{`
        div {
          width: 1em;
          text-align: center;
          display: inline-block;
        }
      `}</style>
    </>
  );
};

export default CharCubeSolved;
