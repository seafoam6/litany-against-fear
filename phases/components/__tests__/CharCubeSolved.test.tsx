import CharCubeSolved from "../CharCubeSolved";
import { render, fireEvent, waitFor, screen } from "@testing-library/react";

describe("CharCubeSolved", () => {
  it("should render", () => {
    const { container, debug } = render(<CharCubeSolved solution="s" />);
    expect(container).toMatchSnapshot();
  });
});
