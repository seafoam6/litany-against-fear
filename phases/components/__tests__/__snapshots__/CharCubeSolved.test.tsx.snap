// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharCubeSolved should render 1`] = `
<div>
  <div
    class="jsx-2087504289"
  >
    s
  </div>
</div>
`;
